### PR TITLE
Prevent double click on graph

### PR DESCRIPTION
**Pull request by lubej**
Double click on ecosystem has odd behaviour, double click navigates directly to Dashboard.

### CODE REVIEW

**Comment by lubej** [comment]
> Can you add instructions to reproduce `double click navigates directly to Dashboard`. I can't reproduce

@lukaw3d unable to reproduce currently as well. Yesterday I was able to reproduce consistently.

**Comment by lubej** [comment]
> > Can you add instructions to reproduce `double click navigates directly to Dashboard`. I can't reproduce
> 
> @lukaw3d unable to reproduce currently as well. Yesterday I was able to reproduce consistently.

@lukaw3d here is the reproduction: https://www.loom.com/share/c043399b62f043e1bd1e3860a020d762

File: src/app/pages/HomePage/Graph/ParaTimeSelector/index.tsx [hunk 1]
```diff
@@ -185,15 +186,21 @@ const ParaTimeSelectorCmp: FC<ParaTimeSelectorProps> = ({ disabled, step, setSte
...
+    if (step !== ParaTimeSelectorStep.Explore) {
+      return
+    }
+
     const transformValue = make3dTransformValue({ x, y, scale })
     setScale(scale)
     quickPinchZoomInnerRef.current?.style.setProperty('transform', transformValue)
   }
 
+  const isZoomedIn = parseFloat(scale.toFixed(2)) > 1
```
Comment (lukaw3d) [inline src/app/pages/HomePage/Graph/ParaTimeSelector/index.tsx:202]: How come `toFixed(2)`?

Comment (lukaw3d) [reply src/app/pages/HomePage/Graph/ParaTimeSelector/index.tsx:202]: This is effectively `scale > 1.005`

**Review by lukaw3d — APPROVED** [review]

File: src/app/pages/HomePage/Graph/ParaTimeSelector/index.tsx (hunk 1 above)
Comment (lubej) [reply src/app/pages/HomePage/Graph/ParaTimeSelector/index.tsx:202]: Initial zoom is something like 1.000000132930. For that purpose I just round to 2 decimals.

**Pull request merged**